app/editor/src/components/upload/Upload.tsx

```diff
@@ -42,17 +44,17 @@ export const Upload: React.FC<IUploadProps> = ({
 
   const [file, setFile] = React.useState<IFile>();
 
+  /** Duration / metadata WIP */
+  // const [duration, setDuration] = useState(0);
+  const reader = new FileReader();
```

Comment:
If possible keep consistent layout for readability.  Variables should be declared if possible before `useEffect(...)` statements.